tests/lifecycle/lifecycle_suite_test.go

```diff
@@ -0,0 +1,75 @@
+package lifecycle
+
+import (
+	"flag"
+	"fmt"
+	"runtime"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+
+	lifecycleparameters "github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/lifeparameters"
```

Comment:
this could be simplified if you change the package name in your parameters.go file to match the folder name (lifeparameters). Or if you wanted it to be called lifecycleparameters you could change the folder name. Either way, you would no longer need to specify the name in the import.